deploy/crds/kieapp.crd.yaml

```diff
@@ -363,9 +362,6 @@ spec:
                               nonXA:
                                 type: string
                                 description: Sets the datasources type. It can be XA or NONXA. For non XA set it to true. Default value is false.
-                              jndiName:
-                                type: string
-                                description: Database JNDI name used by application to resolve the datasource, e.g. java:/jboss/datasources/ExampleDS
```

Comment:
We can't make changes like this to the CRD, specially not properties.

Reply:
Maybe just remove it from the list of required properties and keep it?